oxy/mapping.yaml

```diff
@@ -0,0 +1,43 @@
+1:
```

Comment:
I think we def need to think about how we structure this YAML. It functions as a source of truth of CAN messages on the car and should therefore be extremely human readable and parseable, such that any engineer can look through. I think as we expand, we need to consider both directory structure for within this repo as well as how the actual file is structured. How _most_ companies do this is that they have a ginormous doc of all messages to fully document the protocol (Amazon did this) or have a really big C file that everything includes that defines types (saw this at SimpliSafe, which is the way I originally tried to implement this stuff). TLDR this  is a very important topic to think about moving forward and should not be glossed over